internal/cli/alerts/settings/create.go

```diff
@@ -66,7 +65,20 @@ func CreateBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Creates one alert configuration in the specified project.",
-		Args:  require.NoArgs,
+		Long: `The mongocli atlas alerts settings create command creates
+		one alert configuration in the specified project.`,
+		Example: `  This example uses the mongocli atlas alerts settings create
+		command to create one alert configuration in the specified
+		project. It uses the default profile to access the Atlas project:
+		
+		$ mongocli atlas alert settings create --event JOINED_GROUP --enabled \
+		--notificationType USER --notificationEmailEnabled \
+		--notificationUsername john@example.com \
+		 -o json --projectId 5df90590f10fab5e33de2305`,
+		Annotations: map[string]string{
+			"Args":      "projectId",
+			"projectId": "Project ID to use. Overrides settings from the configuration file or environment variable.",
+		},
```

Comment:
project id is a flag not an arg so this is not needed
```suggestion
```

Reply:
Removed